node/src/chain_spec.rs

```diff
@@ -88,215 +88,10 @@ pub fn development_config() -> Result<ChainSpec, String> {
     ))
 }
 
-pub fn vitalam_config() -> Result<ChainSpec, String> {
-    ChainSpec::from_json_bytes(&include_bytes!("../res/vitalam.json")[..])
-}
-
 pub fn inteli_stage_config() -> Result<ChainSpec, String> {
     ChainSpec::from_json_bytes(&include_bytes!("../res/inteli-stage.json")[..])
```

Comment:
Have not seen `inteli-stage.json` in this PR

Reply:
Indeed, I needed the feature branch to build to regenerate the genesis. That's done and pushed now